.github/scripts/pr_mod_file_tests.py

```diff
@@ -59,28 +59,45 @@ def _file_is_python(filename):
             is_python = True
         else:
             #If no ".py" extension exists, then
-            #open the file and look for a shabang
-            #that contains the word "python".
-            with open(filename, "r", encoding='utf-8') as mod_file:
-                #Loop over lines in file:
-                for line in mod_file:
-
-                    #Ignore blank lines:
-                    if line.strip():
-
-                        #Check that first non-blank
-                        #line is a shabang:
-                        if line[0:2] == '#!':
-                            #If so, then check that the word
-                            #"python" is also present:
-                            if line.find("python") != -1:
-                                #If the word exists, then assume
-                                #it is a python file:
-                                is_python = True
+            #try to open the file and look for
+            #a shabang that contains the word "python".
+            try:
+                with open(filename, "r", encoding='utf-8') as mod_file:
+                    #Loop over lines in file:
+                    for line in mod_file:
+
+                        #Ignore blank lines:
+                        if line.strip():
+
+                            #Check that first non-blank
+                            #line is a shabang:
+                            if line[0:2] == '#!':
+                                #If so, then check that the word
+                                #"python" is also present:
+                                if line.find("python") != -1:
```

Comment:
Probably a little more pedantic but it might be worth modifying these to be more pythonic unless this is hot code and needs as much performance as possible.

```suggestion
                            if line.startswith("#!"):
                                #If so, then check that the word
                                #"python" is also present:
                                if "python" in line:
```

Reply:
Agreed.  I can bring this change in with my commits as well if @K20shores is OK with that.

Reply:
@mwaxmonsky this should be fixed now, but please let me know if that is not the case.